core/idl/wallet/common/wallet.djinni

```diff
@@ -39,10 +39,20 @@ DerivationPath = interface +c {
 
 # Type of operation.
 #
-# It’s possible to either send funds or receive ones.
+# It’s possible to either
+# send funds (getAmount() returns the sent amount),
+# receive funds (getAmount() returns the sent amount),
+# delegate funds (getAmount() returns the *delegated* amount) ; only getFees() are part of balanceHistory
+# undelegate funds (getAmount() returns the *undelegated* amount) ; only getFees() are part of balanceHistory
+# claim rewards (getAmount() returns 0 or the known amount of claimed rewards) ; only getFees() are part of balanceHistory
+#
+# The last 3 operations should not expect to change balanceHistory with their amount, but only their fees.
```

Comment:
Can you also add a small explanation of the `none` situation? Thanks!

Reply:
Will do